### DISCUSSION TITLE
MuTV

### DISCUSSION THREAD

**Discussion by oliver-gomes** [iptv-org/community#662] [Q&A]
**Channel Name:** MuTV
**Channel ID:** MUTV.uk
**Country:** England
**Language:** English
**External Link (optional):** https://mutv.manutd.com/home
**Notes (optional):** xxx

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/uk.m3u#L117-L118
